.github/workflows/sync_from_airtable.yml

```diff
@@ -73,38 +62,48 @@ jobs:
             if (assignees) {
                 const issue  = JSON.parse(process.env.ISSUE);
                 const ghAssignees = issue.data.assignees.map((assignee) => assignee.login);
-                const keepAssignees = []
-                const addAssignees = [];
-                const ghLookup = {};
-                for (let j in ghAssignees) {
-                    ghLookup[ghAssignees[j]] = ghAssignees[j];
-                }
-                for (let i in assignees) {
-                    if (typeof ghLookup[assignees[i]] != 'undefined') {
-                    keepAssignees.push(assignees[i])
-                    delete ghLookup[assignees[i]];
+                const { removeAssignees, addAssignees } = assignees.reduce((acc, labelName, idx) => {
```

Comment:
Nitpick: is there a more expressive name we could use instead of `acc`? Applies to similar callbacks.